membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -126,46 +126,38 @@ class AttributeController(
           )
         } yield {
 
-          def customFields(supporterType: String): List[LogField] = List(
-            LogFieldString("lookup-endpoint-description", endpointDescription),
-            LogFieldString("supporter-type", supporterType),
-            LogFieldString("data-source", fromWhere),
-          )
-
           val result = allProductAttributes match {
             case Some(attrs @ Attributes(_, Some(tier), _, _, _, _, _, _, _, _, _, _, _)) =>
-              logInfoWithCustomFields(
+              logger.info(
```

Comment:
this was sending some extra datapoints to ELK for some purpose,but we haven't sent anything there for about 5 years, so I changed back to the normal logger.info